src/Routes/index.jsx

```diff
@@ -19,7 +19,7 @@ const Router = () => {
       <Route path="/license/search" element={<SearchLicenseWrapper />} />
       <Route path="/system-configuration" element={<FirstRunWrapper />} />
       <Route path={ADMIN_PANEL_PAGE_ROUTE} element={<AdminPanelWrapper />} />
-      <Route path="/key/modify" element={<ModifyProductKeyWrapper />} />
+      <Route path="/license/modify" element={<ModifyLicenseWrapper />} />
```

Comment:
Not as per the latest code. You have removed the upcoming changes during conflicts. Revert